### PR TITLE
Changes to hadder.py and write_fit_input.py

### PR DESCRIPTION
**Pull request by kmei-cms**
Changes to hadder.py and write_fit_input.py and also updated README.md

### CODE REVIEW

File: Analyzer/test/write_fit_input.py [hunk 1]
```diff
@@ -227,26 +227,47 @@ def writeDataCard(self, name):
...
-            "RPV_550" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_550.root",         label="RPV_550", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_650" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_650.root",         label="RPV_650", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_750" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_750.root",         label="RPV_750", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_850" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_850.root",         label="RPV_850", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
+            "SYY_300" : info.DataSetInfo(basedir=basedir, fileName="2016_StealthSYY_2t6j_mStop-300.root", label="SYY_300", processName="signal", process="0", rate=True, lumiSys="1.05", scale=1.17424),
```
Comment (cmadrid1) [inline Analyzer/test/write_fit_input.py]: Every place that you put these new numbers can you do: (newXSec/oldXSec)*options.scale.
This way we still have the option to scale all samples by some common number if we want. I think if that give you problems you might need to change the default options.scale to be equal to 1.0

Comment (cmadrid1) [reply Analyzer/test/write_fit_input.py]: Please test that it works when you do this.

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Last step is to get the cfg and model files for the top tagger and deepESM.
 ```
 cmsenv
 getTaggerCfg.sh -t Tensorflow_Medium_Example_v1.0.2 -o
-getDeepESMCfg.sh -t Keras_Tensorflow_v1.2.5 -o -s 2016
+getDeepESMCfg.sh -t Keras_Tensorflow_v1.2.6 -o -s 2016
```
Comment (cmadrid1) [inline README.md]: This looks good but can you change where this number is used everywhere in the readme? Some of the later commands take this number as a input.

Comment (kmei-cms) [reply README.md]: This should be addressed now

File: Analyzer/test/write_fit_input.py (hunk 1 above)
Comment (kmei-cms) [reply Analyzer/test/write_fit_input.py]: I changed everything you requested and changed the default and tested it (I also tested using the -s option)

**Pull request merged**